### ISSUE TITLE
Screenshots of code in documentation

### ISSUE THREAD

**Issue by osorensen** [DIDSR/WebXR-tools#8]
"TCA Test Instructions" contains several screenshots of code. In my opinion, this way of documenting is not sufficient to satisfy the review item "Functionality documentation". All users may not be able to read code very well. In addition, it will be hard to maintain, since you will have to manually take a new screenshot whenever the code is updated.

Please write out in plain English what the code lines are meant to tell.

https://github.com/openjournals/joss-reviews/issues/6021

**Comment by brendanc490** [comment]
I can see how this would be confusing. The contents in the main branch contain tools that I do not wish to submit, including this TCA test. Only contents of the JOSS submission branch are included in the submission. I can remove this from the main branch if it seems confusing.

**Comment by osorensen** [comment]
Thanks, then I assume this will be fixed once the JOSS branch gets merged onto the main branch.
